docs/rules/authenticity_token.md

```diff
@@ -0,0 +1,65 @@
+# `<input name="authenticity_token">`
```

Comment:
@josh Feel free to commit to this branch if you'd like to change anything in these docs. After we merge this PR, I'll copy the docs into our [security docs](https://github.com/github/github/blob/master/docs/security.md) as well.